sources/packages/web/nginx.conf

```diff
@@ -13,4 +13,10 @@ http {
   default_type  application/octet-stream;
   sendfile        on;
   keepalive_timeout  65;
+  server {
+      add_header 'X-Content-Type-Options'  "nosniff";
+      add_header 'Content-Security-Policy' "default-src 'self'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'";
```

Comment:
The conf file share from other project in story has more headers and more CSP config values. Are we considering it?

Reply:
No currently we are valiating only the medium wava scan issues.